pkg/server/login/locales.go

```diff
@@ -0,0 +1,63 @@
+package login
+
+import "golang.org/x/text/language"
+
+func getLocale(langBase string) map[string]string {
+	switch langBase {
+	case language.English.String():
```

Comment:
Does this work with values like `en-US` and `ja-JP` where a region is used?

Reply:
Yes, that's done by `getPreferredLang()`